api/turing/cmd/main.go

```diff
@@ -122,10 +122,6 @@ func main() {
 	mux.Handle("/v1/internal/", http.StripPrefix("/v1/internal", health))
 	mux.Handle("/v1/", http.StripPrefix("/v1", api.NewRouter(appCtx)))
 
-	if len(cfg.SwaggerFile) > 0 {
-		// Serve Swagger Spec
-		mux.Handle("/openapi.yaml", web.FileHandler(cfg.SwaggerFile, false))
```

Comment:
Do we use `web.FileHandler` anywhere else? If no, shall we drop it?

Reply:
It was used to pass the swagger.yaml file over to the Swaggerui container. So I'm dropping it.

Reply:
I see what you mean. It's used to serve the react files as well.